src/datastore.js

```diff
@@ -200,7 +200,7 @@ s  */
         Buffer.from(this.user + ':' + this.password).toString('base64');
       let url = this.url + 'workspaces/' + workspace + '/coveragestores/' +
         coverageStore + '/file.geotiff';
-      url += '?filename=' + layerTitle + '&coverageName=' + lyrTitle;
+      url += '?filename=' + lyrTitle + '&coverageName=' + layerName;
```

Comment:
I find the name `lyrTitle` confusing, because there is already a `layerTitle`.
For readability it would be nice to use only `layerTitle`

Reply:
By intention since it is no good style to re-write function parameters IMHO.

Reply:
alright.